### PR TITLE
Cleanup after 3679

### PR DESCRIPTION
**Pull request by TracerDS**
Cleanup after #3679

### CODE REVIEW

**Comment by Fernando-A-Rocha** [comment]
Doing other people's work 👹 LGTM

**Comment by TracerDS** [comment]
> Doing other people's work 👹 LGTM

Im just doing gods work 😎

**Review by MegadreamsBE — APPROVED** [review]
You can do cleanup for me anytime. LGTM!

File: Client/game_sa/CEntitySA.cpp [hunk 1]
```diff
@@ -606,101 +606,95 @@ bool CEntitySA::GetBoneRotation(eBone boneId, float& yaw, float& pitch, float& r
...
+    boneOrientation->imag.x = x;
+    boneOrientation->imag.y = y;
+    boneOrientation->imag.z = z;
+    boneOrientation->real = w;
+    CEntitySAInterface* theInterface = GetInterface();
+    if (theInterface)
+        theInterface->bDontUpdateHierarchy = false;
+        
+    return true;
     return false;
```
Comment (FileEX) [inline Client/game_sa/CEntitySA.cpp]: Mistake

**Comment by tederis** [comment]
That's really not necessary. But it's okay, if you wish. The initial PR wasn't perfect but it was good enough if not rolling to the puritanism.

File: Client/game_sa/CEntitySA.cpp (hunk 1 above)
Comment (MegadreamsBE) [reply Client/game_sa/CEntitySA.cpp]: Turns out that my brain is still asleep. Completely missed that in my review

**Comment by TracerDS** [comment]
> That's really not necessary. But it's okay, if you wish. The initial PR wasn't perfect but it was good enough if not rolling to the puritanism.

**No.** It is necessary. We have guidelines set up.

File: Client/game_sa/CEntitySA.cpp (hunk 1 above)
Comment (TracerDS) [reply Client/game_sa/CEntitySA.cpp]: Yeah, I noticed it after merging.

**Comment by tederis** [comment]
> > That's really not necessary. But it's okay, if you wish. The initial PR wasn't perfect but it was good enough if not rolling to the puritanism.
> 
> **No.** It is necessary. We have guidelines set up.

MTA is the real project with its problems and rough edges. You don't need to take the guidelines that serious.

**Comment by TracerDS** [comment]
> > > That's really not necessary. But it's okay, if you wish. The initial PR wasn't perfect but it was good enough if not rolling to the puritanism.
> > 
> > 
> > **No.** It is necessary. We have guidelines set up.
> 
> MTA is the real project with its problems and rough edges. You don't need to take the guidelines that serious.

Im not taking guidelines that serious. Im just following them. Previous PR was just straight up ignoring them, the guy did whatever he pleased. Thats not okay. **Thats exactly why we have guidelines: to follow them**.

If you ignore guidelines then you should not collaborate or chaos would rise again. It was already discussed in discord.

**Comment by tederis** [comment]
> If you ignore guidelines then you should not collaborate or chaos would rise again. It was already discussed in discord.

Small deviations shall be permitted. So please, don't let's argue.

**Comment by TracerDS** [comment]
> > If you ignore guidelines then you should not collaborate or chaos would rise again. It was already discussed in discord.
> 
> Small deviations shall be permitted. So please, don't let's argue.

It wasnt small but whatever you say. Look at his previous PRs and check the comment dutchman posted. Maybe it will clear up your mind.

**Comment by gownosatana** [comment]
The thing you are doing is crazy, in my opinion you shouldn't be even allowed to create/comment on pull requests, it's radicoolous. You are not one about taking decisions here, you argue with TEDERis who is the repo manager which is crazy. Your comments are pointless and have no real reason to be even created, all you do is blocking others pull requests from being merged. I know you know people from MTA staff and that's the only reason you are allowed to do shit like this.

**Comment by TracerDS** [comment]
> The thing you are doing is crazy, in my opinion you shouldn't be even allowed to create/comment on pull requests, it's radicoolous. You are not one about taking decisions here, you argue with TEDERis who is the repo manager which is crazy. Your comments are pointless and have no real reason to be even created, all you do is blocking others pull requests from being merged. I know you know people from MTA staff and that's the only reason you are allowed to do shit like this.

Arguing is a natural thing. Its something all human beings do.
> I know you know people from MTA staff and that's the only reason you are allowed to do shit like this.

You are wrong. I dont know anyone from MTA staff (not in the way you see)

You said it yourself: "you can always create own pull request to refactor code", and so I did.

**Pull request merged**